src/library_wasmfs.js

```diff
@@ -108,12 +108,11 @@ FS.createPreloadedFile = FS_createPreloadedFile;
       });
     },
     // TODO: mkdirTree
-    // TDOO: rmdir
     rmdir: (path) => {
-      return withStackSave(() => {
+      return FS.handleError(withStackSave(() => {
         var buffer = stringToUTF8OnStack(path);
         return __wasmfs_rmdir(buffer);
```

Comment:
Maybe just o this on single line `return withStackSave(() => FS.handleError(__wasmfs_rmdir(stringToUTF8OnStack(path))` ?